verl/utils/checkpoint/checkpoint_manager.py

```diff
@@ -49,10 +49,11 @@ def __init__(
         optimizer: torch.optim.Optimizer,
         lr_scheduler: torch.optim.lr_scheduler.LRScheduler = None,
         processing_class: Union[PreTrainedTokenizer, ProcessorMixin] = None,
-        checkpoint_contents: DictConfig = None,
+        checkpoint_config: DictConfig = None,
```

Comment:
In the next PR, can we use a dataclass instead of DictConfig for `checkpoint_config`?